### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by betzrhodes**
initial library release

### CODE REVIEW

**Comment by betzrhodes** [comment]
opening a new pull request with refactored code

**Comment by betzrhodes** [comment]
Example working.
Tests passing locally.

File: example/sample.agent.nut [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+// EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+// OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+#require "AWSRequestV4.class.nut:1.0.2"
+#require "AWSDynamoDB.agent.lib.nut:1.0.0"
+
+// Enter Your AWS details here
+const AWS_DYNAMO_ACCESS_KEY_ID = "YOUR_KEY_ID_HERE";
```
Comment (ppetrosh) [inline example/sample.agent.nut:29]: Please set up the automated testing here. I'll give you access to the AWS account that we use.

Comment (ppetrosh) [reply example/sample.agent.nut:29]: Please use https://github.com/electricimp/AWSSNS/blob/master/tests/agent.test.nut#L29 for reference. Probably it's worth adding these instructions to the documentation.

**Review by ppetrosh — APPROVED** [review]

**Review by smittytone — APPROVED** [review]

**Pull request merged**